File: java/test/ExistsTest.java
package voxgig.hook0sdk.sdktest;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;

import voxgig.hook0sdk.core.Hook0SDK;

public class ExistsTest {

  @Test
  public void testMode() {
    Hook0SDK testsdk = Hook0SDK.testSDK();
    assertNotNull(testsdk, "expected non-nil SDK");
  }
}

File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Hook0 SDK exists test

require_once __DIR__ . '/../hook0_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = Hook0SDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}
